src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -175,7 +175,9 @@
               (assoc-in [:wallet :ui :send :to-address] to-address)
               (assoc-in [:wallet :ui :send :address-prefix] prefix)
               (assoc-in [:wallet :ui :send :receiver-preferred-networks] receiver-networks)
-              (assoc-in [:wallet :ui :send :receiver-networks] receiver-networks))
+              (assoc-in [:wallet :ui :send :receiver-networks] receiver-networks)
+              (assoc-in [:wallet :ui :send :from-settings?] from-settings?)
```

Comment:
hmm, I'm not such a fan of this data. It seems to be _off_ to couple the navigation starting point to the send data.
I think instead we should just revisit the overall navigation flow from when a user goes from the settings page.
Probably it should immediately close the settings and then initiate the wallet sending flow, i.e that the user can't navigate back to the settings page, but rather can navigate back as far as the app home on wallet tab.

Reply:
i.e this is a design issue at the core.

Reply:
@J-Son89 so you mean it's better to reset the shell navigation to root once the send flow started settings, and if the user exists the send flow will just be back at the current selected tab, correct?

Reply:
yeah that's what I'm thinking. I think it's best to discuss this with designers first though so they're aware of the pattern as there's probably other cases. 

Maybe there's the argument to keep things even simpler and **not** include the send flow there either :)

cc @briansztamfater

Reply:
@J-Son89 I have updated it to reset shell to root when starting send flow from settings:


https://github.com/status-im/status-mobile/assets/29354102/9f1abc7f-47f0-428b-9bff-a1dfa95733e6